txr: 273 -> 274

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/txr/versions.


meta.description for txr is: "Programming language for convenient data munging"

meta.homepage for txr is: "http://nongnu.org/txr"

meta.changelog for txr is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- Warning: no invocation of /nix/store/r6k8vi51hka0bp69h7mhj2fflqs6grfg-txr-274/bin/txr had a zero exit code or showed the expected version
- 0 of 1 passed binary check by having a zero exit code.
- 0 of 1 passed binary check by having the new version present in output.
- found 274 with grep in /nix/store/r6k8vi51hka0bp69h7mhj2fflqs6grfg-txr-274
- found 274 in filename of file in /nix/store/r6k8vi51hka0bp69h7mhj2fflqs6grfg-txr-274

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
0 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
txr
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/r6k8vi51hka0bp69h7mhj2fflqs6grfg-txr-274 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A txr https://github.com/r-ryantm/nixpkgs/archive/845b882d668e8c58842d7c9ef15a133e168bb89b.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/r6k8vi51hka0bp69h7mhj2fflqs6grfg-txr-274
ls -la /nix/store/r6k8vi51hka0bp69h7mhj2fflqs6grfg-txr-274/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>txr</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dtzWill for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
txr: 273 -> 274

dtzWill commented:
Hmm, new segfaults when built against musl? Not a blocker I suppose but would be good to investigate...

dtzWill committed:
txr: latest clang stdenv, fixes w/musl

I have not identified a cause but since this resolves the issue
let's use latest clang for now.

dtzWill committed:
txr: no longer needs bison/flex unless in maintainer mode
